persistence/persistence-mybatis-test/src/test/java/org/cbioportal/persistence/mybatis/MutationMyBatisRepositoryTest.java

```diff
@@ -145,7 +145,6 @@ public void getMutationsWithAll() throws Exception {
 
 		List<Mutation> result = mutationMyBatisRepository.getMutations(sampleIds, entrezGeneIds, 6);
 		Assert.assertEquals(1, result.size());
-		Assert.assertNotNull(result.get(0).getMutationEvent().getGene());
```

Comment:
Since mutationEvent itself isn't a resource that we would expose by itself through an API, it's always part of mutation resource, I removed the Gene property from mutationEvent since mutation class already has it.